packages/@aws-cdk/aws-rds/README.md

```diff
@@ -87,12 +87,15 @@ use the static factory methods on `DatabaseInstanceEngine`:
 ```typescript
 const instance = new rds.DatabaseInstance(this, 'Instance', {
   engine: rds.DatabaseInstanceEngine.oracleSe1({
-    version: '19.0.0.0',
+    version: rds.OracleLegacyEngineVersion.VER_19, // different version class for each engine type
```

Comment:
Change to `oracleSe2` with a non legacy version?

Reply:
Yep. Good catch. Thanks.